python/lsst/validate/drp/validate.py

```diff
@@ -88,17 +92,23 @@ def loadAndMatchData(repo, visitDataIds,
     srcVis = SourceCatalog(newSchema)
 
     for vId in visitDataIds:
-        calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
-        calib.setThrowOnNegativeFlux(False)
+        try:
+            calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
+        except TypeError as te:
+            print(te)
+            continue
```

Comment:
Didn't DM-4133 solve the LTV1/2 mismatched type issue?  Or is this from something else?